pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -153,10 +153,22 @@ func (self *SuggestionsHelper) getRemoteBranchNames(separator string) []string {
 	})
 }
 
+func (self *SuggestionsHelper) getRemoteBranchNamesWithoutRemotePrefix() []string {
+	return slices.FlatMap(self.model.Remotes, func(remote *models.Remote) []string {
+		return slices.Map(remote.Branches, func(branch *models.RemoteBranch) string {
+			return branch.Name
+		})
+	})
+}
+
 func (self *SuggestionsHelper) GetRemoteBranchesSuggestionsFunc(separator string) func(string) []*types.Suggestion {
 	return FuzzySearchFunc(self.getRemoteBranchNames(separator))
 }
 
+func (self *SuggestionsHelper) GetRemoteBranchesWithoutRemotePrefixSuggestionsFunc() func(string) []*types.Suggestion {
```

Comment:
The `ISuggestionsHelper` interface (defined in line 26) doesn't seem to be used (so the compiler doesn't complain when not adding this new function to it). Is this intended? If so, I guess the function should still be added?

Reply:
It is used at line 41, but since you didn't add any new functions to the interface itself the compiler's got nothing to complain about. 

_If_ you'd added a function to the interface, but without an implementation, _that_ would raise up an error.

Reply:
Apologies for some interfaces that aren't actually being used. The intention was to use them in tests but I haven't gotten around to actually setting up some exemplar tests (I'm still mostly depending on integration tests which have proven pretty robust).

With that said feel free to delete `ISuggestionsHelper` if it's not being used.

Reply:
Ok. I'd keep them for now to keep the PR as small as possible.